src/pages/Vault/index.tsx

```diff
@@ -1,8 +1,49 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
+import React, { useContext } from 'react';
 
-const VaultUi: React.FC = () => <div>Vault</div>;
+import { Spinner } from 'components';
+import { AuthContext } from 'context/AuthContext';
+import { useGetVaults } from 'clients/api';
+import { Vault } from 'types';
+import VaultItem from './VaultItem';
+import { useStyles } from './styles';
 
-const Vault: React.FC = () => <VaultUi />;
+export interface IVaultUi {
+  vaults: Vault[];
+}
 
-export default Vault;
+export const VaultUi: React.FC<IVaultUi> = ({ vaults }) => {
+  const styles = useStyles();
+
+  if (vaults.length === 0) {
+    return <Spinner />;
+  }
+
+  return (
+    <div css={styles.container}>
+      {vaults.map(vault => (
```

Comment:
We also need to add the VAI and  VRT vaults. I didn't see a separate ticket for those.

Reply:
Looks like the VAI vault is in another PR so I assume the VRT will follow